openlibrary/components/BarcodeScanner/components/LazyBookCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <a class="lazy-book-card" :href="link">
```

Comment:
Add the loading class to show the loading indicator

```suggestion
  <a class="lazy-book-card" :href="link" :class="{ 'loading': loading }">
```

Reply:
Oh, I think we might need something similar for showing the errored indicator!